scripts/soldoc.sh

```diff
@@ -0,0 +1,11 @@
+mkdir docs .flattened
+rm docs/*.md
+for f in contracts/*.sol
```

Comment:
We should use something like `for f in $(find contracts -name *.sol)` which would walk through the nested contracts (such as in `deployment`) as well. We could probably ignore contracts we don't want to be generated (such as maybe `LifTokenTest`) as well with find.